src/views/NewPatient.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div id="newPatient">
+    <section id="cart">
+      <section id="close">
+        <button @click="closeNotice" id="btnClose">
+          <i class="far fa-times-circle"></i>
+        </button>
+        <div id="icon-hospital-user">
+          <i class="fas fa-hospital-user"></i>
+        </div>
+        <span class="title-new-patient">new patient</span>
+      </section>
+      <section id="start">
+        <div class="patient-id">
+          <h4>Patient ID</h4>
+          <p>MUM9033_ISS2n_FAC132</p>
+          <h4>Adress</h4>
+          <p>Utangwa</p>
+          <h4>Coordinates</h4>
+          <p>
+            1° 42' 48.7" S <br />
+            37° 26' 41.0" E
+          </p>
+        </div>
+        <primaryButton
+          @click="toPickUp"
+          class="start-travel-btn"
+          :text="'start travel'"
+        />
+      </section>
+    </section>
+  </div>
+</template>
+
+<script>
+import primaryButton from "../components/globalComp/primaryButton";
+export default {
+  name: "NewPatient",
+  components: { primaryButton },
+  methods: {
+    toPickUp() {
+      this.$router.push("/pickup");
+    },
+    closeNotice() {
+      this.$emit("closeNotice");
```

Comment:
Får detta felmeddelandet: "45:18  error  Custom event name 'closeNotice' must be kebab-case". Antar att du bara behöver skriva ett bindestreck mellan close och notice och sen ha små bokstäver.